#No input
---
string upper
string lower
string outer
string shoes